validation and qualification/README.md

```diff
@@ -0,0 +1,23 @@
+### Unit and integration tests
```

Comment:
Linked to all unit test on AppVeyor for Tools and components
Also linked to subfolders containing Ranorex/Comparison/... Reports